src/TransportTests/ConfigureAzureServiceBusTransportInfrastructure.cs

```diff
@@ -19,7 +19,7 @@ public TransportConfigurationResult Configure(SettingsHolder settings, Transport
 
         var transportExtensions = new TransportExtensions<AzureServiceBusTransport>(settings);
 
-        transportExtensions.SubscriptionNameShortener(name =>
+        transportExtensions.SubscriptionNameConvention(name =>
```

Comment:
Do we want the hashed values to be used on everything in the tests? Or perhaps we should a length calculation here?

Reply:
Good catch. It should check for the length similar to what other places are doing as we used to use the shortening API.